pyproject.toml

```diff
@@ -376,8 +359,15 @@ lint.ignore = [  # NOTE: non-permanent exclusions should be added to `.ruff.toml
 
     # Ruff-specific rules (RUF)
     "RUF005",  # unpack-instead-of-concatenating-to-collection-literal -- it's not clearly faster.
+
+    # ISC001 shouldn't be used with ruff format
+    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+    "ISC001",
```

Comment:
It's best to keep the list of ignored rules in alphabetical order. Admittedly RUF005 is in the wrong place too, but this pull request shouldn't worry about that (and I'm not convinced that RUF005 belongs on this list anyways).

Reply:
Done! Well I would argue as soon as we want to keep the things in order here, we should have a formatter tool :)
Time for pyproject-fmt?

Reply:
#15365  👀